### PR TITLE
Add osbuild platform

### PR DESCRIPTION
**Pull request by evgenyz**
#### Description:

- This is a platform for OSBuild pipeline.
- It is based on the value of the `container` environment variable (`bwrap-osbuild`).

#### Rationale:

- This platform is a prerequisite for fixing: https://github.com/ComplianceAsCode/content/issues/9536

### CODE REVIEW

File: shared/checks/oval/installed_env_is_osbuild.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+       </affected>
+      <description>Check the value of environment variable container.</description>
+      <reference ref_id="cpe:/a:osbuild" source="CPE" />
+    </metadata>
+    <criteria>
+      <criterion comment="Check the value of container variable" test_ref="test_installed_env_is_osbuild" />
+    </criteria>
+  </definition>
+
+  <ind:environmentvariable58_test check_existence="all_exist"
```
Comment (vojtapolasek) [inline shared/checks/oval/installed_env_is_osbuild.xml:16]:
```suggestion
  <ind:environmentvariable58_test check="all" check_existence="all_exist"
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello @evgenyz  and thank you for the new platform. Please fix the OVAL as suggested, that should make  tests happy.

File: shared/checks/oval/installed_env_is_osbuild.xml (hunk 1 above)
Comment (evgenyz) [reply shared/checks/oval/installed_env_is_osbuild.xml:16]: Yeah, thanks.

**Comment by evgenyz** [comment]
This platform has absolutely nothing to do with OCP, we can ignore it.

**Review by vojtapolasek — APPROVED** [review]
Thank you for the new platform. The CI failures seem to happen due to infrastructure problems. Also the platform should not affect the OCP in any way... mainly because it is not used in the content yet.

**Pull request merged**